colossus-tests/src/test/scala/colossus/protocols/http/client/StreamingHttpResponseParserSpec.scala

```diff
@@ -97,6 +99,45 @@ class StreamingHttpResponseParserSpec extends WordSpec with MustMatchers with Tr
 
       callbackExecuted must equal (true)
     }
+    "decode a response that was encoded by colossus with no body" in {
+      validateStreamingEncoding(ByteString(""))
+    }
+
+    "decode a response that was encoded by colossus with a body" in {
+      validateStreamingEncoding(ByteString("body bytes!"))
+    }
+
   }
 
+  private def validateStreamingEncoding(body : ByteString){
```

Comment:
This test just tests that when we encode a streamed response, we can successfully decode it, which should verify that it is a valid http response.